src/SOS/SOS.UnitTests/SOS.UnitTests.csproj

```diff
@@ -44,6 +44,7 @@
 <![CDATA[
 <Configuration>
   <TargetConfiguration>$(Configuration)</TargetConfiguration>
+  <RepoRootDir>$(RepoRoot)</RepoRootDir>
```

Comment:
Should we try [MSBuild]::NormalizePath on this?

Reply:
It is already normalized:

<Configuration>
  <TargetConfiguration>Debug</TargetConfiguration>
  <RepoRootDir>C:\ssd\diagnostics\</RepoRootDir>
  <CDBPath>C:\Users\mikem\.nuget\packages\cdb-sos\10.0.26100.1\runtimes\win-$(TargetArchitecture)\native\cdb.exe</CDBPath>
</Configuration>